backend/people/migrations/0020_fill_workshops.py

```diff
@@ -0,0 +1,78 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.26 on 2020-10-22 19:47
+from __future__ import unicode_literals
+from django.db import migrations
+
```

Comment:
The workshop model is in the `events` app. So, this migration must be in the events app.

Reply:
Done